### DISCUSSION TITLE
Topical Statuspage updates

### DISCUSSION THREAD

**Discussion by jamiemccarthy** [community/community#48071] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

For my company, I've subscribed to githubstatus.com so we get updates in our "deploys" Slack channel. Unfortunately it's very noisy.

To start with, we get hourly updates about issues with GitHub systems that we don't care about for the purposes of deploying. We only care about Git Operations, API Requests, Pull Requests, and Actions. We don't need updates on Pages, Codespaces, Copilot, and the rest in our deploy channel.

There are also cases where multiple updates are sent unnecessarily. On Friday night we got 13 "Investigating" Slack updates in 8 minutes, all with virtually the same information.

![Screenshot 2023-02-20 at 12 21 53 PM](https://user-images.githubusercontent.com/40373/220177382-7d4493e4-68ce-431c-bd82-93d51cbce9bf.png)

For this purpose we don't need the hourly updates. Every six hours, maybe.

Finally, if an incident is confined to a region of the world we're not in, we'd love to not see that either.

Could you publish RSS feeds that slice up status updates in such a way that we could subscribe to exactly what we want? Then we could just use Slack's RSS app to post them to our channel. Thank you.

**Comment by bdmartin** [comment]
FWIW, Statuspage allows for subscriptions by components, it just needs to be enabled by GitHub.

https://support.atlassian.com/statuspage/docs/enable-component-subscriptions/

**Comment by mattm-trustedhealth** [comment]
PLEASEEEEEEEEEEEE enable this. Someone, please

**Comment by Jared-Stensland** [comment]
It would be very helpful to only subscribe to the components that we use
